modules/core/arrow-typeclasses/src/main/kotlin/arrow/typeclasses/internal/internal.kt

```diff
@@ -0,0 +1,25 @@
+package arrow.typeclasses.internal
+
+import arrow.Kind
+import arrow.core.Either
+import arrow.core.ForId
+import arrow.core.Id
+import arrow.core.fix
+import arrow.typeclasses.Monad
+
+val IdMonad: Monad<ForId> = object : Monad<ForId> {
```

Comment:
Why not just:
```kotlin
val Id.Companion.monad: Monad<ForId> = object : Monad<ForId> { ... }
```

Reply:
Why duplicating it you mean? To have all instances on the same place. This one we want to control because you won't be able to override it.